model-optimizer/mo/utils/extract_release_version.py

```diff
@@ -0,0 +1,42 @@
+"""
+ Copyright (C) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import re
+
+try:
+    # needed by install_prerequisites which call extract_release_version as python script
+    from version import get_mo_version
+except ImportError:
+    from mo.utils.version import get_mo_version
+
+
+def extract_release_version():
+    version = get_mo_version()
+    patterns = [
+        # captures release version set by CI for example: '2021.1.0-1028-55e4d5673a8'
+        "^([0-9]+).([0-9]+)*",
+        # captures release version generated by MO from release branch, for example: 'custom_releases/2021/1_55e4d567'
+        "_releases/([0-9]+)/([0-9]+)_*"
+    ]
+
+    for pattern in patterns:
+        m = re.search(pattern, version)
+        if m:
```

Comment:
Need additionally check that there are at least 2 matched items. Because if there is just one then `m.group(2)` will result in an exception.

Reply:
added.